lib/checkingaccount.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'account.rb'
+module Bank
+  class CheckingAccount < Account
+    attr_reader :count
+
+    def initialize(id = "", balance = 10, date_opened = "" )
+      @count = 0
+      super(id, balance, date_opened)
+      if balance < 10
+        raise ArgumentError.new("balance must be >= 10")
+        puts "balance must be >= 10"
+      end
+    end
+
+    def withdraw(amount,limit= 10)
```

Comment:
I like how your design of `Account#withdraw` makes this method and `withdraw_using_check` both very clean. Good work!